### ISSUE TITLE
occurrences in French Guiana

### ISSUE THREAD

**Issue by BdeThoisy** [darrennorris/TACAR#1]
Darren
I have concern regarding the presence in FG. Podocnemis DO NOT occur on the west of Cayenne
The more western population are on the bassin of the Mahury river (close East of Cayenne). 

And, to my opinion, all the population of the Oyapock river (frontier FG/Amapa) is under threat, it is the more highly poached river

Benoit

**Comment by BdeThoisy** [comment]
indeed, there are people in the upper limit of this river. Maybe not on the data base, but hundreds of amerindians there ("Trois Sauts village")

**Comment by darrennorris** [comment]
thanks @BdeThoisy Benoit for taking time to look into the mapped distribution. 

1) Are you able to provide references that support the absences to the west of Cayenne? Is this a result of human developments e.g. they were once there originallly but have now been extirpated (or perhaps still occur at very low densities)?  Any published data for rivers that drain into Maroni River / Suriname? It is simple to exclude basins.....,but would be important to have evidences to do so.

2)The analysis is conservative (best-case), so I am expecting there to be areas where the predictions underestimate true impacts.  
The overall result for French Guiana is Vulnerable-Endangered (see figure in the results section: 
https://darrennorris.github.io/TACAR/articles/a02_Matrix-population-model-projections.html#findings). 
Does this seem reasonable based on the results presented? Would you consider the more pessimistic Endangered assessment more plausible?

3)Thank you for the info on Oyapock river / ("Trois Sauts village"). The map only shows a categorical classification where populations are Endangered (reduction > 50% in three generations) or not -  I was trying to keep the map clear. Would an additional map be usefull that shows the population change values on a continuous scale?

**Comment by darrennorris** [comment]
Hi Benoit @BdeThoisy ,
Please check the updated version here: https://darrennorris.github.io/TACAR/articles/a01_Interactive-map.html
I am using HydroBasin level six to include/exclude river basins ( https://www.hydrosheds.org/products/hydrobasins ). 
If there remain finer scale local variations this could be worked on in the future as part of a more detailed national level aseessment.
Please let me know.
Best,
Darren
